### PR TITLE
Add Uninstall Doc

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: docs/install/uninstall.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```bash
+curl -sL https://raw.githubusercontent.com/rancher/rke2/488bab0f48b848e408ce399c32e7f5f73ce96129/bundle/bin/rke2-killall.sh --output rke2-killall.sh
+chmod +x rke2-killall.sh
+mv rke2-killall.sh /usr/local/bin
+```
+
+Now run the `rke2-uninstall.sh` script. This will call the `rke2-killall.sh` script which will 
+
+```bash
+./rke2-uninstall.sh
```
Comment (ShylajaDevadiga) [inline docs/install/uninstall.md:32]: Can we replace ./rke2-unsintall.sh by rke2-unsintall.sh

File: docs/install/uninstall.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+After the `rke2-uninstall.sh` and `rke2-killall.sh` scripts complete,run the commands below to perform additional clean uup.
+
+```bash
+yum remove -y rke2-*;
```
Comment (ShylajaDevadiga) [inline docs/install/uninstall.md:38]: Can we also remove ; in the end?

File: docs/install/uninstall.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Now run the `rke2-uninstall.sh` script. This will call the `rke2-killall.sh` script which will 
+
+```bash
+rke2-uninstall.sh
+```
+
+After the `rke2-uninstall.sh` and `rke2-killall.sh` scripts complete,run the commands below to perform additional clean uup.
+
+```bash
+yum remove -y rke2-*
```
Comment (ShylajaDevadiga) [inline docs/install/uninstall.md:38]: Run this before calling rke2-uninstall.sh

**Comment by ShylajaDevadiga** [comment]
LGTM

File: docs/install/uninstall.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+---
+title: Uninstall
+---
+
+# Uninstall
```
Comment (cjellick) [inline docs/install/uninstall.md:5]:
```suggestion
# Uninstall

This document explains how to fully uninstall RKE2 on a node, based on the installation method that was used.
```

File: docs/install/uninstall.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+To uninstall RKE2 from your system, simply run the command below. This will shutdown process, remove the RKE2 binary, and clean up files used by RKE2.
+
+```bash
+rke2-uninstall.sh
+```
+
+## RPM Method
+
+To uninstll RKE2 from your system, if installed with from RPM, a few commands need to be ran.
```
Comment (cjellick) [inline docs/install/uninstall.md:17]:
```suggestion
To uninstll RKE2 from your system, if installed with from RPM, a few commands need to be ran. 

> **Note:** RPM based installs currently do not install the rke2-uninstall.sh script. This is a known issue that will be addressed in a future release. This document instructs you on how to download and use the necessary scripts.
```

**Review by cjellick — APPROVED** [review]

**Pull request merged**